src/Router/StorybooksRouter.tsx

```diff
@@ -33,11 +38,15 @@ export class StorybooksRouter extends React.Component<Props> {
   }
 
   render() {
-    const { ClientApp } = this.state
+    const ClientApp = this.state && this.state.ClientApp
 
     return (
       <React.Fragment>
-        {ClientApp && <ClientApp {...this.props.initialState} />}
+        {ClientApp && (
+          <ClientApp subscribeTo={this.props.subscribeTo}>
+            {this.props.children}
+          </ClientApp>
```

Comment:
Having said the above (about passing all props in one location), I guess these could move there too.